docker/Dockerfile_stack

```diff
@@ -47,12 +47,7 @@ RUN apt-get update && apt-get install -y \
     rm -rf /var/lib/apt/lists/*
 
 RUN pip install --upgrade --no-cache-dir pip && \
-    pip install --no-cache-dir sphinx==1.4 sphinx_rtd_theme breathe==4.6 && \
-    echo "deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-4.0 main" > /etc/apt/sources.list.d/llvm.list && \
```

Comment:
So we dont have to do all this for 5.0 anymore?

Reply:
This was old copy/paste from Dockerfile_clang-format and the LLVM part ended being duplicated in this file.